socorro/external/es/crashstorage.py

```diff
@@ -319,6 +320,26 @@ def capture_crash_metrics(self, raw_crash, processed_crash):
             # later.
             self.config.logger.exception('something went wrong when capturing processed_crash_size')
 
+    def _index_crash(self, connection, es_index, es_doctype, crash_document, crash_id):
+        try:
+            start_time = time.time()
+            connection.index(
+                index=es_index,
+                doc_type=es_doctype,
+                body=crash_document,
+                id=crash_id
+            )
+            index_outcome = 'successful'
+        except Exception:
+            index_outcome = 'failed'
+            raise
+        finally:
+            elapsed_time = time.time() - start_time
+            self.metrics.histogram(
+                '%s_index' % index_outcome,
```

Comment:
I'm second guessing myself from a few months ago now that willkg got me thinking about metric names. Maybe this should just be named `index` and have a tag that indicates success or failure?

Whichever we do here, we should do the same for s3.

Reply:
Yeah... +1 for putting the outcome in a tag.